repair/row_level.cc

```diff
@@ -2003,7 +2003,9 @@ static future<> repair_get_row_diff_with_rpc_stream_process_op_slow_path(
             co_await sink(repair_row_on_wire_with_cmd{repair_stream_cmd::end_of_current_rows, repair_row_on_wire()});
         } else {
             for (repair_row_on_wire& row : rows_on_wire) {
-                co_await sink(repair_row_on_wire_with_cmd{repair_stream_cmd::row_data, std::move(row)});
+                auto cmd = repair_row_on_wire_with_cmd{repair_stream_cmd::row_data, std::move(row)};
+                co_await sink(cmd);
+                co_await cmd.row.clear_gently();
```

Comment:
I see. repair_row_on_wire is misnamed, because it encompasses multiple rows.

Isn't it a problem to send on rpc if it can contain many rows?

Reply:
Yes, each repair_row_on_wire could contain multiple fragments within the same partition. 
Since we have repair_rows_on_wire, that's why we used  repair_row_on_wire for partition_key_and_mutation_fragments.
```
using repair_row_on_wire = partition_key_and_mutation_fragments;
using repair_rows_on_wire = std::list<partition_key_and_mutation_fragments>;
```
The data is sent over rpc stream not a single rpc verb call.